src/SUMMARY.md

```diff
@@ -23,5 +23,8 @@
     - [C8Y Mapper Operation Handling](software-management/c8y-mapper-operation-handling.md)
     - [Thin Edge JSON Mapping to/from C8Y](software-management/thin-edge-json-mapping-to-from-c8y.md)
 
+## Self-Update
+- [Self-Update](self-update/README.md)
+
```

Comment:
I would put `Self-Update` under `Software management` rather than adding a new section.

Reply:
Yes, good point. I will move it.